### PR TITLE
add download button

### LINKED ISSUE #84: Allow downloading of spec file
The app doesn't currently allow a user to download the spec file.

It should allow users to do this. It could look like below.

image

### PR DESCRIPTION
**Pull request by quangkhoa**
resolve #84

### CODE REVIEW

File: src/components/DownloadButton/DownloadButton.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { PropTypes, PureComponent } from 'react'
+import './DownloadButton.scss'
+
+export default class DownloadButton extends PureComponent {
+  render () {
+    return (
+      <a className='download-button' target='_blank' href={this.props.url}>Download</a>
```
Comment (brendo) [inline src/components/DownloadButton/DownloadButton.js]: Please add `rel="noopener noreferrer"`, see [this article](https://mathiasbynens.github.io/rel-noopener/).

File: src/components/DownloadButton/DownloadButton.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.download-button {
+  //border: 1px solid rgb(132, 150, 168);
```
Comment (brendo) [inline src/components/DownloadButton/DownloadButton.scss]: Remove commented out code.

**Review by brendo — CHANGES_REQUESTED** [review]

File: src/components/DownloadButton/DownloadButton.js (hunk 1 above)
Comment (quangkhoa) [reply src/components/DownloadButton/DownloadButton.js]: Thank you Brendan. Added.

File: src/components/DownloadButton/DownloadButton.scss (hunk 2 above)
Comment (quangkhoa) [reply src/components/DownloadButton/DownloadButton.scss]: Removed.

**Review by brendo — APPROVED** [review]

**Pull request merged**